ampersand-array-input-view.js

```diff
@@ -183,8 +174,15 @@ module.exports = View.extend({
         this.fields = without(this.fields, field);
         field.remove();
         this.fieldsRendered -= 1;
+        this._setRemoveableFields();
         this.update();
     },
+    _setRemoveableFields: function () {
+        var removable = this.fields.length > (this.minLength || 1);
```

Comment:
I'd argue that the condition should simply be `this.fields.length > this.minLength` (without a minimum of 1), as a minLength of zero would suggest that all fields are technically removable.

I kept the `|| 1` to mimic previous behaviour (see L153), but my recommendation would be to remove it.